consensus/types/src/light_client_update.rs

```diff
@@ -86,79 +91,20 @@ pub struct LightClientUpdate<T: EthSpec> {
     pub signature_slot: Slot,
 }
 
-impl<T: EthSpec> LightClientUpdate<T> {
-    pub fn new(
-        chain_spec: ChainSpec,
-        beacon_state: BeaconState<T>,
-        block: BeaconBlock<T>,
-        attested_state: &mut BeaconState<T>,
-        finalized_block: BeaconBlock<T>,
-    ) -> Result<Self, Error> {
```

Comment:
oops sorry I think this one might actually be needed for now (used in one of the other light client update PR) - we're only computing and caching `LightClientOptimisticUpdate` and `LightClientFinalityUpdate` in [`LightClientServerCache`](https://github.com/sigp/lighthouse/blob/b23c913605ccb4d565a3f687377954aaac6b7d47/beacon_node/beacon_chain/src/light_client_server_cache.rs#L27) at the moment.

Reply:
brought this back